RELEASE_NOTES.md

```diff
@@ -1,3 +1,45 @@
+## Cookstyle 6.17
+
+### 4 New Cops
+
+#### ChefDeprecations/ChefShellout
+
+The `ChefDeprecations/ChefShellout` cop detects the use of the `Chef::ShellOut` class which was removed in Chef Infra Client 13. It will autocorrect all occurences to instead use `Mixlib::ShellOut`, which is a drop-in replacement.
```

Comment:
```suggestion
The `ChefDeprecations/ChefShellout` cop detects the use of the `Chef::ShellOut` class which was removed in Chef Infra Client 13. It will autocorrect all occurrences to instead use `Mixlib::ShellOut`, which is a drop-in replacement.
```

Reply:
Fixed